composer.json

```diff
@@ -11,16 +11,21 @@
         }
     ],
     "require": {
-        "php": "^5.4 || ^7.0",
-        "php-http/httplug": "^1.1",
+        "php": "^7.0",
```

Comment:
why not go directly to 7.1? afaik 7.0 is EOL already.

Reply:
Because it's the same of PSR-18 and HTTPlug 2.

Reply:
Im fine with 7.1. For psr18 it did not really matter. It was not using any php7.1 features.

Reply:
as long as we agree that dropping 7.0 support is a minor change and we "just do it" if 7.0 becomes a pain for some reason, i am fine with allowing it for now ;-)